### PR TITLE
libssh: update to libssh-0.8.1

### PR DESCRIPTION
**Pull request by vpeter4**
make vfs.sftp kodi binary addon working
with previous version kodi crashed on start
tested with Generic build on VMware Workstation Pro

### CODE REVIEW

**Comment by MilhouseVH** [comment]
Presumably this is not dependent on #2887?

**Comment by vpeter4** [comment]
Well, I bumped this package after #2887 applied. But I think it doesn't matter because vfs.sftp addon is broken for some time now.

File: packages/network/libssh/package.mk [hunk 1]
```diff
@@ -2,12 +2,12 @@
 # Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
 
 PKG_NAME="libssh"
-PKG_VERSION="0.7.5"
-PKG_SHA256="6f7d5b9f3089164eec7408d35b99194b17352cce436a4a95485b511bce2aab4b"
+PKG_VERSION="0.8.1"
+PKG_SHA256="d17f1267b4a5e46c0fbe66d39a3e702b8cefe788928f2eb6e339a18bb00b1924"
 PKG_ARCH="any"
 PKG_LICENSE="OpenSource"
```
Comment (CvH) [inline packages/network/libssh/package.mk]: could you pls change it to `PKG_LICENSE="BSD"`

Comment (vpeter4) [reply packages/network/libssh/package.mk]: You mean LGPL as mentioned at https://www.libssh.org/features/ ?

**Comment by vpeter4** [comment]
If libssh package is build after krb5 then there is undefined symbol GSS_C_NT_HOSTBASED_SERVICE in  vfs.sftp addon. One option is to disable gssapi in libssh package with _-DWITH_GSSAPI=OFF_. The other options is in krb5. No idea what is better but worth to think on.

**Pull request merged**